lib/amqp/Vhost.js

```diff
@@ -322,7 +323,29 @@ function Vhost(config) {
         return channel.close();
       }
 
-      once(null, channel);
+      channel.consume(
+        'amq.rabbitmq.reply-to',
+        function(message) {
+          const { correlationId } = message.properties;
+
+          const replyHandler = self.replyHandlers[correlationId];
+
+          if (!replyHandler) return;
+
+          let content = message.content.toString();
+
+          try {
+            content = JSON.parse(content);
+          } catch (e) {}
```

Comment:
Assuming we follow the normal approach fur subscriptions, we should emit appropriate errors rather than swallow them